statistics/statistics.css

```diff
@@ -3,8 +3,28 @@ margin: 0;
 padding: 0;
 box-sizing: border-box;
 }
+/*START DARK THEME SECTION */
+:root {
+    --bg-c: #fffAF4;
+    --total-cases-text:#FF8A23;
+    --note-p:rgba(0, 0, 0, 0.6);
+    --bg-cont-card:#FFF1E1;
+    --t-btn:#FF8A23;
+}
+.dark {
+    --bg-c: #3d5a80;
+    --total-cases-text:#e0fbfc;
+    --note-p:#e0fbfc;
+    --bg-cont-card:#e0fbfc;
+    --t-btn:#ffdeeb;
+}
+.toggle-btn{
+    padding-left:10%;
+    color: var(--t-btn);
+}
+/*END DARK THEME SECTION */
 body {
-background: #fffAF4;
+background: var(--bg-c); /* FOR DARK THEME */
```

Comment:
Alwasy add browser fall back and debugging fallback when you use `var()`
Check the links below
[Attach a Fallback value to a CSS Variable](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/attach-a-fallback-value-to-a-css-variable)
[Improve Compatibility with Browser Fallbacks](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/improve-compatibility-with-browser-fallbacks)

Note: Browser fallback much more important and it considered as enough its alone